exercises/jade/problem.md

```diff
@@ -49,3 +49,5 @@ without the time.
 When creating your projects from scratch, install the `jade` dependency with npm.
 
 Again, the port to use is passed by {appname} to the application as `process.argv[2]`.
+
+If you receive `Error: Cannot find module 'jade'`, it is because Express is looking for Jade relative to its path. You can fix this by running `npm install Jade'.
```

Comment:
can you fix `npm install Jade` into `npm install jade`? and fix the closing '. it should be ` not '